common/next/get-prismic-global-alert.js

```diff
@@ -0,0 +1,18 @@
+const Prismic = require('prismic-javascript');
+let globalAlert = {
+  text: [ { type: 'paragraph',
+    text: null,
+    spans: null } ],
+  isShown: 'hide'
+};
+async function getAndSetGlobalAlert() {
+  try {
```

Comment:
Been wondering about if we could try a higher order utility for async/await error handling (something I saw in a [Wes Bos talk](https://wesbos.github.io/Async-Await-Talk/#57)).

`const tryPromise = (fn, errHandler) => (...args) => fn(...args).catch(errHandler || console.error);`

Thoughts?

Reply:
Error handling could default to `Raven.captureException(err)` maybe